src/main/java/com/hibernate/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.hibernate.service.impl;
+
+import com.hibernate.dao.UserDao;
+import com.hibernate.dao.impl.UserDaoImpl;
+import com.hibernate.lib.Inject;
+import com.hibernate.lib.Service;
+import com.hibernate.model.User;
+import com.hibernate.service.UserService;
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Inject
+    private UserDao userDao = new UserDaoImpl();
+
+    @Override
+    public User add(User user) {
```

Comment:
```suggestion
    public User add(User user) {
```
You should create salt and hash password in UserService add() method.